packages/c/cumem/xmake.lua

```diff
@@ -0,0 +1,30 @@
+package("cumem")
+    set_kind("library", { headeronly = true })
+    set_description("CUDA Memory Management Wrapper with Type Safety")
+    set_homepage("https://github.com/BinhaoQin/cuMem")
+    add_urls("https://github.com/BinhaoQin/cuMem/archive/refs/tags/$(version).tar.gz")
+    add_versions("1.0.0", "1c7956c840f7aa3940756c8d09a2ae44ce53681ae01f3378cf4501d81bf0638c")
+    add_deps("cuda", { system = true })
+
+    on_install(function(package)
```

Comment:
you need limit the supported patforms.

`on_install("linux", "windows", function`